homeassistant/components/amberelectric/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/amberelectric",
   "iot_class": "cloud_polling",
   "loggers": ["amberelectric"],
-  "requirements": ["amberelectric==1.1.1"]
+  "requirements": ["amberelectric==2.0.1"]
```

Comment:
You're still using a caret in the version specifier for pydantic in pyproject.toml. That means it won't allow version 2+.